manifests/init.pp

```diff
@@ -225,6 +227,7 @@
   Optional[String] $db_username = $::foreman::params::db_username,
   Optional[String] $db_password = $::foreman::params::db_password,
   Optional[String] $db_sslmode = 'UNSET',
+  Optional[String] $db_root_cert = 'UNSET',
```

Comment:
I dislike the ```UNSET```. It used to be needed in puppet versions but now I think we can just use ```undef```.